internal/internal_workflow.go

```diff
@@ -1436,6 +1546,29 @@ func (h *queryHandler) execute(input []interface{}) (result interface{}, err err
 	return executeFunction(h.fn, input)
 }
 
+func (h *updateHandler) validate(input []interface{}) (err error) {
+	defer func() {
+		if p := recover(); p != nil {
+			st := getStackTraceRaw("update validator [panic]:", 7, 0)
+			err = fmt.Errorf("update validator panic: %v, stack trace: %v", p, st)
```

Comment:
I think you should reuse the `PanicError` in this package, if not for other reasons, just to have that `PanicError` type on the application failure info same as we do with other panics.

Reply:
No problem :+1: